packages/apps/human-app/frontend/src/api/services/operator/disable-operator.ts

```diff
@@ -21,8 +22,9 @@ export function useDisableWeb3Operator() {
         },
       });
 
-      await mutateAsync({ authType: 'web3' });
+      await mutateAsyncIfNotLoading({ authType: 'web3' });
```

Comment:
Here and in other places, why not imported name `refreshAccessTokenAsync`/`refreshAccessToken`?
Using it as `mutate` is confusing

Reply:
Sure, will fix that, I used auto rename in IDE